pkg/features/kube_features.go

```diff
@@ -192,14 +160,6 @@ const (
 	// Normalize HttpGet URL and Header passing for lifecycle handlers with probers.
 	ConsistentHTTPGetHandlers featuregate.Feature = "ConsistentHTTPGetHandlers"
```

Comment:
Why is this not removed?

Reply:
CI always failed when I removed this feature-gate. I try to figure out why it failed